browser-test/src/admin/admin_program_creation.test.ts

```diff
@@ -510,6 +510,39 @@ test.describe('program creation', () => {
     await expect(addressCorrectionInput).toHaveValue('false')
   })
 
+  test('create program with markdown questions', async ({
+    page,
+    adminQuestions,
+    adminPrograms,
+  }) => {
+    await loginAsAdmin(page)
+    await adminQuestions.addTextQuestion({
```

Comment:
suggestion (blocking): Add text for the other fields where we expect markdown to show up (I think this might just be `helpText`, but I'm not positive)

Reply:
I think this comment is not resolved yet. My suggestion was to add `helpText` with markdown in it so we can cover that use case as well.

Reply:
This is still not resolved. My suggestion is to add `helpText` with markdown in it.

Reply:
woops, missed this comment!